src/Php72/Php72.php

```diff
@@ -80,4 +83,36 @@ public static function php_os_family()
 
         return isset($map[PHP_OS]) ? $map[PHP_OS] : 'Unknown';
     }
+
+    public static function spl_object_id($object)
+    {
+        if (null === self::$hashMask) {
+            self::initHashMask();
+        }
+
+        return self::$hashMask ^ hexdec(substr(spl_object_hash($object), self::$hashOffset, PHP_INT_SIZE));
+    }
+
+    private static function initHashMask()
+    {
+        $obj = (object) array();
+        self::$hashOffset = 16 - PHP_INT_SIZE;
```

Comment:
if you want to improve perf even further, you could even fully qualify these constants (as done for `\PHP_VERSION_ID`). Not sure it is worth it though (this code is called only once)